lighthouse/helpers/reports.py

```diff
@@ -191,26 +190,34 @@ def get_locations_from_labwhere(labware_barcodes):
     )
 
 
-def get_cherrypicked_samples(root_sample_ids):
+def get_cherrypicked_samples(root_sample_ids, plate_barcodes):
     # Find which samples have been cherrypicked using MLWH & Events warehouse
     # Returns dataframe with 1 column, 'Root Sample ID', containing Root Sample ID of those that have been cherrypicked
+    # TODO: move into external method.
```

Comment:
It would be nice to move the SQL into its own method. And then a comment on the method explaining that the query is doing